src/physics/initial_conditions/initial_condition.h

```diff
@@ -151,6 +151,24 @@ class InitialConditionFunction_BurgersViscous: public InitialConditionFunction<d
     real value (const dealii::Point<dim,real> &point, const unsigned int istate = 0) const override;
 };
 
+/// Initial Condition Function: 1D Sine Function; used for temporal convergence
+template <int dim, int nstate, typename real>
+class InitialConditionFunction_1DSine
+        : public InitialConditionFunction<dim,nstate,real>
+{
+protected:
+    using dealii::Function<dim,real>::value; ///< dealii::Function we are templating on
+
+public:
+    /// Constructor for InitialConditionFunction_BurgersRewienski
```

Comment:
update comment for doxygen

Reply:
Changed, as well as in `exact_solution.h`.